cip/CIP2015-10-12-CREATE.adoc

```diff
@@ -0,0 +1,136 @@
+= CIP2015-10-12 - CREATE
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Authors:* Pontus Melke pontus.melke@neotechnology.com, Mats Rydberg mats@neotechnology.com
+
+[abstract]
+.Abstract
+--
+This proposal formalizes `CREATE` which is used in Cypher to create nodes and relationships.
```

Comment:
Does/should this cover `CREATE UNIQUE` as well (which is part of the same parser rule in the Neo4j implementation)? Similarly, might be also worth mentioning that `CREATE` is used in other contexts within Cypher, such as `CREATE CONSTRAINT`, for clarity.